### PR TITLE
Fix modal navigation bar

### PR DESCRIPTION
**Pull request by marianeum**
Turns out it was wrong after all. Also update screenshots to be edge to edge to be correct.

+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: .idea/compiler.xml
Comment (marianeum) [inline .idea/compiler.xml:1]: post gradle update change

File: backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/modal/BpkModal.kt [hunk 1]
```diff
@@ -79,6 +79,7 @@ fun BpkModal(
             if (activityWindow != null && dialogWindow != null) {
                 if (!isSystemInDarkTheme) {
                     WindowInsetsControllerCompat(dialogWindow, dialogWindow.decorView).isAppearanceLightStatusBars = true
+                    WindowInsetsControllerCompat(dialogWindow, dialogWindow.decorView).isAppearanceLightNavigationBars = true
```
Comment (marianeum) [inline backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/modal/BpkModal.kt:82]: this is the fix for the modal

File: .idea/compiler.xml
Comment (alejandrorosas) [reply .idea/compiler.xml:1]: weird that it's 21 for intellij but we have 17 setup everywhere 😅

**Review by alejandrorosas — APPROVED** [review]

File: .idea/compiler.xml
Comment (marianeum) [reply .idea/compiler.xml:1]: I know - i undid this and it came back again, so I think it really wants it 🤷‍♀️

**Pull request merged**